browser/sources.gni

```diff
@@ -174,6 +174,7 @@ brave_chrome_browser_deps = [
   "//brave/components/cosmetic_filters/common:mojom",
   "//brave/components/de_amp/browser",
   "//brave/components/de_amp/common",
+  "//brave/components/debounce/content/browser",
```

Comment:
why is this needed here now? I would have expected to see an existing dep for `//brave/components/debounce/browser` if we need this

Reply:
oic, this should go in `brave_browser_debounce_deps` above

Reply:
oic, brave_browser_debounce_deps handled the deps for the factory which previously included the throttle